sass/directives/_headers.scss

```diff
@@ -172,3 +172,28 @@
     }
   }
 }
+
+@mixin header--small {
+  max-height: 300px;
+
+  &__headline {
+    padding: $header-padding 0;
+    max-width: $header-width-normal;
+ 
+    h1 {
```

Comment:
these margins should probably be set in an existing or a new pattern from https://github.com/cb-talent-development/employer-style-base/blob/master/sass/directives/_headings.scss